test/com/sun/jna/ELFAnalyserTest.java

```diff
@@ -104,5 +116,16 @@ private static void extractFileFromZip(File zipTarget, String zipEntryName, File
             zip.close();
         }
     }
+
+    private static void makeLinuxArmelNoflagLib(File sourceFile, File outputFile) throws IOException {
+        final int POS_ABI_FLOAT_BIT = (byte) 0x25;
```

Comment:
A comment would be helpful here to understand why this works. Something about the lines:
The e_flags for elf arm binaries begin at an offset of 0x24 bytes. The procedure call standard is coded on the second byte.

Reply:
ok, thank you for your advice.

Reply:
thank you for your comments and just a moment please.
i will fix your comment asap.